source/php-frameworks/drupal.txt

```diff
@@ -0,0 +1,110 @@
+.. _php-drupal-integration:
+
+==========================
+Drupal MongoDB Integration
+==========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: php framework, CMS, web app, authentication, Atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn about the benefits of using MongoDB Atlas
+as the database for Drupal sites and navigate to a tutorial that
+demonstrates how to set up the MongoDB driver for Drupal.
+
+`Drupal <https://www.drupal.org/about>`__ is an open-source web content
+management system (CMS) written in PHP. Drupal provides simple content
+authoring, composability, and robust authentication features.
+
+The :ref:`php-drupal-tutorial` section of this guide links to a tutorial
+which you can follow to set up a Drupal site that connects to MongoDB Atlas.
```

Comment:
I: [that/which](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-that:~:text=Use%20that%20to%20introduce%20a%20restrictive%20clause%2C)

```suggestion
that you can follow to set up a Drupal site that connects to MongoDB Atlas.
```

Reply:
this one always gets me